src/MLJGLMInterface.jl

```diff
@@ -291,17 +291,29 @@ function glm_report(glm_model, features, reportkeys)
 end
 
 """
-    glm_formula(model, features) -> FormulaTerm
+    glm_formula(model, features::AbstractVector{Symbol}) -> FormulaTerm
 
 Return formula which is ready to be passed to `fit(form, data, ...)`.
 """
-function glm_formula(model, features)::FormulaTerm
+function glm_formula(model, features::AbstractVector{Symbol})::FormulaTerm
     # By default, using a JuliaStats formula will add an intercept.
     # Adding a zero term explicitly disables the intercept.
     # See the StatsModels.jl tests for more information.
     intercept_term = model.fit_intercept ? 1 : 0
     form = GLM.Term(:y) ~ sum(GLM.term.(features)) + GLM.term(intercept_term)
```

Comment:
instead you can do

```julia
form = GLM.StatsModels.FormulaTerm(GLM.Term(:y), sum(GLM.term.(features)) + GLM.term(intercept_term)))
```

then you don't need to do the cleanup.

the sorting of terms is only triggered by the `~` operator, not the constructor:

```julia
julia> term(:y) ~ term(:a) + term(:b) + term(1)
FormulaTerm
Response:
  y(unknown)
Predictors:
  1
  a(unknown)
  b(unknown)

julia> FormulaTerm(term(:y), term(:a) + term(:b) + term(1))
FormulaTerm
Response:
  y(unknown)
Predictors:
  a(unknown)
  b(unknown)
  1
```